frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/selectors.ts

```diff
@@ -0,0 +1,31 @@
+import { getName } from '@console/shared/src';
+import { TemplateKind } from '@console/internal/module/k8s';
+import {
+  getTemplatesOfLabelType,
+  getTemplatesWithLabels,
+} from '../../../../selectors/vm-template/advanced';
+import {
+  getFlavorLabel,
+  getOsLabel,
+  getWorkloadLabel,
+} from '../../../../selectors/vm-template/combined-dependent';
+import { TEMPLATE_TYPE_BASE } from '../../../../constants/vm';
+
+export const findTemplate = (
```

Comment:
can we use a `FindTemplateParams` here, somthing liks ?
``` ts
type FindTemplateParams = {
  templates: TemplateKind[],
  {
    userTemplateName: string,
  } | {
    workload: string,
    flavor: string,
    os: string,
 }
}
```

Reply:
created the type for options

Reply:
just making it a type ... I think that you do not need os, flavor and workload if you have the template ... ?